docs/INTERNAL.md

```diff
@@ -1,45 +0,0 @@
-# Elastic Internal Documentation
```

Comment:
Are we sure we don't want this anymore? These kinds of guides can be useful for new folks when they join.

Reply:
I can add it back.